tests/tests_bridge.yml

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: BSD-3-Clause
 ---
```

Comment:
Please remove these changes from this PR.

Reply:
done